src/sa_web/static/js/handlebars-helpers.js

```diff
@@ -160,21 +172,35 @@ var Shareabouts = Shareabouts || {};
   });
 
   Handlebars.registerHelper('each_place_item', function() {
-    var result = '',
+    var self = this,
+        result = '',
         args = Array.prototype.slice.call(arguments),
         exclusions, options;
 
     options = args.slice(-1)[0];
     exclusions = args.slice(0, args.length-1);
+    
+    // check if a report has been made with the dynamic form, or with the regular form, and iterate appropriately
+    _.each(this.from_dynamic_form ? NS.Config.place.categories[this.location_type].fields : NS.Config.place.items, function(item, i) {      
```

Comment:
I'm not sure if we want to be tracking the dynamic form within the model itself because it is mixing our model's state with the state of our view. 

If a solution doesn't come to mind soon, we can always refactor this after the merge.